scripts/run_kickstart_tests.sh

```diff
@@ -279,9 +295,13 @@ for t in ${tests}; do
     else
         ks=${t/.sh/.ks.in}
     fi
+    tests_ks+="${ks/.ks.in/.ks} "
     sed ${sed_args} ${ks} > ${t/.sh/.ks}
 done
 
+# do %ksappend substition on all the files as well
```

Comment:
Doesn't the commit message say that ksappend substitution comes before the ks.in -> ks substitution?